handlers.go

```diff
@@ -359,3 +427,21 @@ func isHistoryStale(history *ecrpc.PairData, threshold time.Duration) bool {
 	// to the configured threshold duration.
 	return time.Unix(recentTimestamp, 0).Before(time.Now().Add(-threshold))
 }
+
+// mergePairData merges the pair data from two pairs based on data freshness.
```

Comment:
Nit: freshness => most recent timestamp ?

Reply:
Thanks, that is more clear instead of `freshness`!